Resources/Prototypes/_Goobstation/Catalog/Fills/Crates/syndicate.yml

```
@@ -60,6 +60,16 @@
       - id: DoubleEmergencyNitrogenTankFilled
 
 - type: entity
+  parent: CrateSyndicate
+  id: CrateSyndicateBingle
+  name: tigercoorp B1-NG bundle
```

first, typo, secondly i think it should just be B1-NG bundle.
```suggestion
  name: B1-NG bundle